### ISSUE TITLE
to early to difficult

### ISSUE THREAD

**Issue by Mannshoch** [JSandomierz/pcis#3]
May the fan could blow a bit lower in the first part.
May you could add an easy mode where the gravitation is a bit lower.
if I cross the ball wit the bar  I draw, it could give him a pushing instead crossing the ball.

**Comment by RicoElectrico** [comment]
Yeah. That's my feeling too. The gameplay could probably use more balance.

For such a simple concept the game has quite good mechanics that emerge from it, like drawing platforms below "down" arrows, which allows you to gain considerable speed. Likewise, converting sideways speed from a fan (or horizontal powerup) to upwards speed.

But it's hard to get any higher than 100-200 m other than by pure luck. This is because you simply can't control Polandball _if_ you ever happen to actually gain speed. Going at a speed of few screens per second you can lose by stumbling into a random Britainball or Russiaball you couldn't have seen.

Some random ideas to make gameplay more fun:
* Don't spawn Russiaball / Britainball if speed is too high. For instance, choose points in some rectangle (say 50% or 100% or whatever of the screen height) up over the current screen viewport and spawn them only if they happen to be out of screen for longer than X ms.
* Give Polandball more lives so he doesn't die on single hit.
* Plot Polandball future trajectory so it's easier to draw platforms (maybe invent easy mode for this?)

**Comment by run-dey** [comment]
>converting sideways speed from a fan (or horizontal powerup) to upwards speed

Yes, this is the trick. Except you need to build horizontal speed not only from the fans/the wind, but from repeatedly bouncing PB off the edges of the screen (notwithstanding the fact that it doesn't make sense in-game). I even deflect him right at the start of the game for this purpose.

>it's hard to get any higher than 100-200 m other than by pure luck

Not really. My high score is 533 after a couple of hours (using the above technique) and I reached 450+ twice too. But I'll grant you that IIRC 2/3 of those games ended in a superspeed collision just like you described. Maybe there are other ways to increase difficulty at higher altitudes. Maybe making the bouncers less visible (perhaps by making them black), so that visual feedback for the player is reduced. Or maybe some other game physics concept can be included which the player will have to control as well. Maybe some kind of orbital bodies or radiation can be introduced which will create an extra factor in PB's trajectories, which the player will have to increasingly account for, instead of risking unavoidable collisions.

**Comment by baccyflap** [comment]
I really enjoy the game as it is right now, where you sometimes get screwed over by the level. It's part of the fun that you can suddenly get two Britain balls right in the middle of the screen that you have to maneuver around carefully, or that you get some sideways-throw-things next to a Russia ball and stuff like that. And if you start another game, there will be different hard parts.

But to each their own, of course, so a secondary mode with the features you suggest could be a good idea for people who would like to play the game in a different way. Another idea for that: temporary invincibility that can be triggered a set number of times.

**Comment by Mannshoch** [comment]
as a possible solution for the Problems @RicoElectrico faced. Maybe if you gain enough speed there could appear a kind of radar-control at the top that warn about a barrier ahead.
